maintainers/maintainer-list.nix

```diff
@@ -11360,6 +11360,11 @@
     githubId = 357005;
     name = "Marco Orovecchia";
   };
+  orthros = {
```

Comment:
Should be in a separate commit.

Reply:
Sounds good!

Just for my clarity: a separate commit in this PR, or a separate commit in a second PR?